### PR TITLE
solved test #117

### LINKED ISSUE #117: (SP: 1) Write unit test for "Find offers" page
### Page unit test
Unit test for "**Find offers**" page
Scenaries descriptions:
- [ ] It should render page
- [ ] Text of the page must be in the document

[Link to component](https://github.com/Space2Study-Project/Space2Study-FrontEnd/blob/develop/src/pages/find-offers/FindOffers.jsx)

Current coverage:
Image

### PR DESCRIPTION
**Pull request by AndriiNasadiuk**
close #117 
Знімок екрана 2023-11-30 о 17 26 31

### CODE REVIEW

**Review by marianamartsy — APPROVED** [review]
👍

**Review by LeonidShamarin — APPROVED** [review]

**Review by ArtemHolikov — APPROVED** [review]
Nice, yes. Correct

File: tests/unit/pages/find-offers/FindOffers.spec.jsx
Comment (vlad-khrychov) [inline tests/unit/pages/find-offers/FindOffers.spec.jsx:1]: In task was described 2 scenarios, we need at least 2 `expect` (or 2 `it`)
image

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]

File: tests/unit/pages/find-offers/FindOffers.spec.jsx
Comment (AndriiNasadiuk) [reply tests/unit/pages/find-offers/FindOffers.spec.jsx:1]: ok, I’ll do the second

Comment (AndriiNasadiuk) [reply tests/unit/pages/find-offers/FindOffers.spec.jsx:1]: I have done it

**Review by Mygal-Alex — APPROVED** [review]

**Review by vlad-khrychov — APPROVED** [review]

**Pull request merged**